drug-interaction/cli/drug-interaction.psl

```diff
@@ -0,0 +1,12 @@
+1: ATCSimilarity(D1, D2) & interacts(D1, D3) & validInteraction(D1, D3) & validInteraction(D2, D3) & (D2 - D3) & (D1 - D2)>>interacts(D2, D3) ^2
```

Comment:
Use `->` instead of `>>`.
And make sure to always include a space around binary operators (except commas).